tests/test_face.py

```diff
@@ -10,13 +13,60 @@ def test_face(self):
         data_folder = self.test_folder() / "test_data"
         self.run_test_on_all_files_in_folder(data_folder)
 
+    def test_get_triangles(self):
+        """ Test getting triangles from a face """
+        # Test a single block
+        block_file = self.test_folder() / "test_data/block.step"
+        solids = list(Compound.load_from_step(block_file).solids())
+        for solid in solids:
+            solid.triangulate_all_faces()
+            faces = solid.faces()
+            for face in faces:
+                # Check getting triangles without normals
+                verts, tris = face.get_triangles()
+                self.assertIsInstance(verts, np.ndarray)
+                self.assertIsInstance(tris, np.ndarray)
+                self.assertEqual(verts.shape[1], 3)
+                self.assertEqual(tris.shape[1], 3)
+                # Check getting triangles with normals
+                verts, tris, normals = face.get_triangles(return_normals=True)
+                self.assertIsInstance(normals, np.ndarray)
+                self.assertEqual(normals.shape[1], 3)

```

Comment:
Perhaps an assert to make sure number of vertices and number of normals match along dim 0?

Reply:
Done!